docs/docs-content/clusters/edge/trusted-boot/edgeforge/check-efi-limit.md

```diff
@@ -27,16 +27,26 @@ ISO file.
   - Minimum RAM: 8
   - Minimum Storage: 100 GB
 
-- Your Edge host supports Unified Extensible Firmware Interface (UEFI) boot options.
+- The ability to modify the boot order settings to boot from a USB drive and to enter the BIOS interface. Typically this
+  requires you to have a wired keyboard and a monitor to display the interface connected to the Edge device.
 
-- You have already built the installer ISO file that you will use to install Palette on your Edge host. For more
-  information, refer to [Build Installer ISO with Trusted Boot](./build-trusted-iso.md).
+- Your Edge host supports UEFI boot options.
+
+- You have already built the installer ISO file as well as the provider image that you will use to provision clusters on
+  your Edge host. For more information, refer to [EdgeForge with Trusted Boot](../edgeforge/edgeforge.md).
 
 - You can only perform the actions in this guide **prior to** enrolling keys in your Edge host. The ISO that is used to
   check the EFI boot limit will not have your keys on it. If your device has already enrolled your custom keys and have
   secure boot enabled, the Edge host will not be able to boot from the unsigned volume.
 
-## Check Bootable EFI Size Limit for Hardware
+- [Git](https://cli.github.com/manual/installation). You can ensure git installation by issuing the git --version
```

Comment:
```suggestion
- [Git](https://www.git-scm.com/downloads). You can ensure git installation by issuing the git --version
```

https://cli.github.com/manual/installation is a broken URL. I always use the official URL from git: https://www.git-scm.com/downloads